README.md

```diff
@@ -1,175 +1,302 @@
-    Django version 1.11.4, using settings 'usaspending_api.settings'
-    Starting development server at http://127.0.0.1:8000/
-    Quit the server with CONTROL-C.
-
+```
+Performing system checks...
+System check identified no issues (0 silenced).
+August 22, 2017 - 15:55:33
+Django version 1.11.4, using settings 'usaspending_api.settings'
+Starting development server at http://127.0.0.1:8000/
+Quit the server with CONTROL-C.
+```
 Now, go to `http://localhost:8000/api/v1/awards/` to see the API! There aren't any other urls set up at the moment.
 
 ## Running the Test Suite
 
 To run the test suite and confirm that everything is working as expected, make sure you're in the top-level project directory and that your `DATABASE_URL` environment variable is set, and
-run the following from the command line:
 
-
-        py.test
+run the following from the command line:
+```
+$ py.test
+```
 
 The tests will not run against your database in `$DATABASE_URL`, but against a
+
 temporary database named `test_(your db name)`, which they will create.
 
 ## ETL Setup
 
 The Django server has extract, transform, and load (ETL) processes built in as custom management commands. To use them, follow these setup steps:
 
-  1. Configure the `DATA_BROKER_DATABASE_URL` environment variable. This is the same process as setting up the API backend. Exporting `DATA_BROKER_DATABASE_URL` should look like this:
+  
 
-  ```
-  $ export DATA_BROKER_DATABASE_URL='postgres://USER:PASSWORD@HOST:PORT/DATABASENAME'
-  ```
+1. Configure the `DATA_BROKER_DATABASE_URL` environment variable. This is the same process as setting up the API backend. Exporting `DATA_BROKER_DATABASE_URL` should look like this:
+
+```
+$ export DATA_BROKER_DATABASE_URL='postgres://USER:PASSWORD@HOST:PORT/DATABASENAME'
+```
 
 ## Loading Data
 
+  
+
 For details on loading reference data, DATA Act Broker submissions, and current USAspending data into the API, see [loading_data.md](loading_data.md).
 
+  
+
 For details on how our data loaders modify incoming data, see [data_changes.md](data_changes.md).
 
+  
+
 ## Management Commands
 
+  
+
 Below is a list of available management commands a brief description of their uses. For more help, check the command's help text.
 
-* `load_reference_data` - Loads all reference data (used on a fresh or flushed db)
-* `update_location_usage_flags` - Updates the `place_of_performance_flag` and `recipient_flag` on Location objects based upon their foreign key usages
-* `load_submission` - Loads a submission from the broker for the currently set environment
-* `loadtas` - Loads tas account information from a file
-* `loadagencies` - Loads agency information from a file
-* `loadcfda` - Loads CFDA information from a file
-* `loadcontracts` - Loads USASpending contract information from a file
-* `generate_test_endpoint_responses` - Generates endpoint test responses for testing
+  
+*  `load_reference_data` - Loads all reference data (used on a fresh or flushed db)
```

Comment:
We have a number of management commands. I don't know if it makes sense to draw attention to any of them in this README.

Reply:
resolved in new readme